lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -190,13 +190,12 @@ uint32_t SessionStateEnforcer::update_session_rules(
       case ACCESS:
         // Get new UPF TEID
         if (upf_teid_get) {
-          upf_teid = insert_pdr_from_access(session_state, rule, session_uc);
+          upf_teid = insert_pdr_from_access(session, rule, session_uc);
         }
         break;
       case CORE:
         if (gnb_teid_get) {
-          if (!insert_pdr_from_core(imsi, session_state, rule, session_uc))
-            return 0;
+          if (!insert_pdr_from_core(imsi, session, rule, session_uc)) return 0;

```

Comment:
Can we add a bracket around the if clause here?

Reply:
Added braces for readability to if condition.